change/@microsoft-teams-js-9e642ffe-0bf6-4775-a92f-99a31e897e32.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
```

Comment:
>major [](http://example.com/codeflow?start=11&length=5)

This doesn't seem like a major change. It doesn't break any existing APIs and there's I don't think there's anything an existing consumer would have to do to integrate with the newer version of this library (e.g. they wouldn't have to change their code)

Reply:
Modified, thank you for pointing that out.